src/Microsoft.AspNet.Razor/RazorEditorParser.cs

```diff
@@ -61,11 +86,13 @@ public class RazorEditorParser : IDisposable
         private Block _currentParseTree;
 
         /// <summary>
-        /// Constructs the editor parser.  One instance should be used per active editor.  This
-        /// instance _can_ be shared among reparses, but should _never_ be shared between documents.
+        /// Constructs the editor parser. One instance should be used per active editor. This
+        /// instance <em>can</em> be shared among reparses, but should <em>never</em> be shared between documents.
         /// </summary>
-        /// <param name="host">The <see cref="RazorEngineHost"/> which defines the environment in which the generated code will live.  <see cref="F:RazorEngineHost.DesignTimeMode"/> should be set if design-time code mappings are desired</param>
-        /// <param name="sourceFileName">The physical path to use in line pragmas</param>
+        /// <param name="host">The <see cref="RazorEngineHost"/> which defines the environment in which the generated
+        /// code will live. <see cref="RazorEngineHost.DesignTimeMode"/> should be set if design-time code mappings
```

Comment:
`<see cref="RazorEngineHost.DesignTimeMode"/> should be set if design-time code mappings are desired.` => `<see cref="RazorEngineHost.DesignTimeMode"/> should be set if design-time behavior is desired.`